dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2175,33 +2422,40 @@
       "x-ms-discriminator-value": "AreaResolution",
       "properties": {
         "unit": {
-          "type": "string",
-          "enum": [
-            "Unspecified",
-            "SquareKilometer",
-            "SquareHectometer",
-            "SquareDecameter",
-            "SquareDecimeter",
-            "SquareMeter",
-            "SquareCentimeter",
-            "SquareMillimeter",
-            "SquareInch",
-            "SquareFoot",
-            "SquareMile",
-            "SquareYard",
-            "Acre"
-          ],
-          "x-ms-enum": {
-            "name": "AreaUnit",
-            "modelAsString": true
-          },
-          "description": "The area Unit of measurement"
+          "$ref": "#/definitions/AreaUnit"
         }
       },
       "required": [
         "unit"
       ]
     },
+    "LengthUnit": {
+      "type": "string",
+      "enum": [
+        "Unspecified",
+        "Kilometer",
+        "Hectometer",
+        "Decameter",
+        "Meter",
+        "Decimeter",
+        "Centimeter",
+        "Millimeter",
+        "Micrometer",
+        "Nanometer",
+        "Picometer",
+        "Mile",
+        "Yard",
+        "Inch",
+        "Foot",
+        "LightYear",
+        "Pt"
```

Comment:
All the others are spelled out. Why not "Point"?

Reply:
It looks like this has been addressed, so I will mark it as resolved.